specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.generatedetailedcostreport.json

```diff
@@ -0,0 +1,373 @@
+          },
+          "GenerateDetailedCostReportByBillingAccountLegacyAndBillingPeriod": {
+            "$ref": "./examples/GenerateDetailedCostReportByBillingAccountLegacyAndBillingPeriod.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeUsageDetailsParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GenerateDetailedCostReportDefinition"
+            },
+            "description": "Parameters supplied to the Create detailed cost report operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request processing completed.",
+            "schema": {
+              "$ref": "#/definitions/GenerateDetailedCostReportOperationResult"
+            }
+          },
+          "202": {
+            "description": "Accepted. Request will be processed. Use the Location header to check the status.",
+            "headers": {
+              "Location": {
+                "description": "The URL to check the result of the asynchronous operation.",
+                "type": "string"
+              },
+              "Azure-Consumption-AsyncOperation": {
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Azure-AsyncOperation": {
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "format": "int32",
+                "type": "integer"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/GenerateDetailedCostReportErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.CostManagement/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "GenerateDetailedCostReportOperationResults"
+        ],
+        "operationId": "GenerateDetailedCostReportOperationResults_Get",
+        "description": "Get the result of the specified operation. This link is provided in the GenerateDetailedCostReport creation request response header.",
```

Comment:
>This link is provided in the GenerateDetailedCostReport creation request response header.", [](http://example.com/codeflow?start=67&length=91)

Since this looks like a general purpose operation result API, I'd reword this to something like 

"Gets the result of the specified operation. The link with this operationId is provided as a response header of the initial request.",